examples/simple/.envrc

```diff
@@ -0,0 +1,2 @@
+#Override defaults
+command -v dotenv && test -f .env && dotenv
```

Comment:
do we need this file or you pushed it by a mistake?

Reply:
its ok